src/atoms/View/View.web.js

```diff
@@ -1,3 +1,18 @@
-const View = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const View = styled(({ as, children, ...props }) => React.createElement(as, props, children))`
+  ${''}
+`;
+
+View.propTypes = {
+  as: PropTypes.string.isRequired,
```

Comment:
```suggestion
  as: PropTypes.oneOf(['div', 'p', 'span']).isRequired,
```

add more values to it if you want but accept only specific values that you want to allow

Reply:
@deorukhkarchaitanya let it be string. We'll restrict if we see the issue because on web it has lot of nuances and then the lead time to get the PR merged is hugeee.

@susheelachoudry Remove the `isRequired` though